inventory/any.cf

```diff
@@ -622,7 +622,7 @@ bundle common cfe_autorun_inventory_aws
         expression => isreadable("/sys/hypervisor/uuid", 1);
 @else
       "sys_hypervisor_uuid_readable" -> { "ENT-9931" }
-        expression => returnszero("${paths.cat} /sys/hypervisor/uuid 2>&1 >/dev/null", "useshell");
+        expression => returnszero("${paths.cat} /sys/hypervisor/uuid >/dev/null 2>&1", "useshell");
```

Comment:
I have no opinion really, but you prefer no space after the first `>` ?

Reply:
Also, `${}` is a bit a-typical for MPF which usually uses `$()`.

Reply:
I think that CFEngine would skip the promise that contained an unresolved variable.

Reply:
indeed it would!